templates/zerver/help/preview-your-message-before-sending.md

```diff
@@ -1,11 +1,15 @@
 # Previewing your message
 
-You can preview your message before you send it by clicking the **eye** (![eye](/static/images/help/eye.png)) icon.
+You can preview your message before you send it by clicking the **eye**
+(![eye](/static/images/help/eye.png)) icon at the bottom of the **Compose message box**.
 
 ![Original content](/static/images/help/before-preview.png)
 
+This is bring up the message preview:
```

Comment:
You can format this as "2. After clicking the eye (insert image) icon, the message composer will be replaced with a preview of your message" or something similar, since that's an incomplete sentence and we want the docs to be formatted as a series of steps.